scrapy_zyte_api/handler.py

```diff
@@ -76,11 +76,11 @@ def _prepare_api_params(self, request: Request) -> Optional[dict]:
         api_params: Dict[str, Any] = self._zyte_api_default_params or {}
         try:
             api_params.update(meta_params)
-        except TypeError:
+        except ValueError:
```

Comment:
I think you might need to catch both. ValueError is raised in a test because dict.update does support lists, but they should be lists of tuples. If you pass e.g. `123`, a TypeError is raised.

Reply:
Good catch! https://github.com/scrapy-plugins/scrapy-zyte-api/pull/42/commits/7b61e981304742d8a81d1b4ee14f9d56f853b31c